components/nexus-repository/src/main/java/org/sonatype/nexus/repository/proxy/ProxyFacetSupport.java

```diff
@@ -86,6 +86,8 @@ public String toString() {
 
   private boolean remoteUrlChanged;
 
+  private String cacheToken;
```

Comment:
As discussed in the call this needs to be persisted, but perhaps that's another PR.